### PR TITLE
RUBY-2933 Fix getMore with comment specs

### PR DESCRIPTION
**Pull request by comandeo**
https://jira.mongodb.org/browse/RUBY-2912
https://jira.mongodb.org/browse/RUBY-2933
https://jira.mongodb.org/browse/RUBY-2934

### CODE REVIEW

File: lib/mongo/cursor.rb [hunk 1]
```diff
@@ -424,12 +424,10 @@ def get_more_operation
...
         end,
-        comment: if view.respond_to?(:options) && view.options.is_a?(Hash)
-          view.options[:comment]
-        else
-          nil
-        end,
       }
+      if view.respond_to?(:options) && view.options.is_a?(Hash)
+        spec[:comment] = view.options[:comment] unless view.options[:comment].nil?
+      end
```
Comment (Neilshweky) [inline lib/mongo/cursor.rb:430]: This might be a dumb question, but why don't you check for `get_more_comment_enabled?` here as well?

**Review by Neilshweky — APPROVED** [review]

File: lib/mongo/cursor.rb (hunk 1 above)
Comment (comandeo) [reply lib/mongo/cursor.rb:430]: We need a `connection` object to do the feature check, but we do not have it here, unfortunately.

Comment (Neilshweky) [reply lib/mongo/cursor.rb:430]: I see, and we don't get the connection until much later... Is this going to lead to unnecessarily sending comments when the feature is not enabled?

Comment (comandeo) [reply lib/mongo/cursor.rb:430]: It shouldn't. We check if the feature is enabled on the operation layer, when we do have a connection - https://github.com/mongodb/mongo-ruby-driver/pull/2432/files#diff-091c8fb9685f4516c8ef985fd2e1abe701fa8c45bf94c44103159f073e905649R34
I do not think there is a chance that a `comment` will be send if the feature is not enabled.

Comment (Neilshweky) [reply lib/mongo/cursor.rb:430]: okay I understand now, thanks for the explanation!

**Review by p-mongo — APPROVED** [review]

**Pull request merged**